dashboard/app/controllers/registrations_controller.rb

```diff
@@ -8,6 +8,8 @@
 require 'queries/lti'
 
 class RegistrationsController < Devise::RegistrationsController
+  before_action :require_no_authentication, only: [:account_type, :login_type, :finish_student_account, :finish_teacher_account]
```

Comment:
I am genuinely unsure here but do we need to add `new`, etc to this list? ie does this still work for the existing flow?

Reply:
Silly old me for thinking a test would catch this... added the inherited calls in explicitly and added test coverage!

Reply:
:lolsob: thanks for adding the extra test coverage!